documentation/docs/reference/pmm_components_and_versions.md

```diff
@@ -2,20 +2,20 @@
 
 The following table lists all the PMM client/server components and their versions:
 
-| PMM client/server component |Version | Documentation |Location on GitHub|
-|-----------------------------|--------|---------------|------------------|
-| Grafana  | 11.1.8*    | [Grafana Documentation](https://grafana.com/docs/grafana/latest/)|[Github Grafana](https://github.com/percona-platform/grafana)|
-| VictoriaMetrics| 1.93.4    | [VictoriaMetrics Documentation](https://docs.victoriametrics.com/)|[Github VictoriaMetrics](https://github.com/VictoriaMetrics/VictoriaMetrics)    |
-| Nginx    | 1.20.1 | [Nginx Documentation](http://nginx.org/en/docs/)|[Github Nginx](https://github.com/nginx/nginx)                                                    |
-| Percona Distribution for PostgreSQL  | 14.5    | [Percona Distribution for PostgreSQL 14 Documentation](https://www.percona.com/doc/postgresql/LATEST/index.html)|              |
-| Clickhouse| 23.8.2.7 |[ClickHouse Documentation Documentation](https://clickhouse.com/docs/en/)|[Github ClickHouse](https://github.com/ClickHouse/ClickHouse)|
-| PerconaToolkit  | 3.4.0    | [Percona Toolkit Documentation](https://www.percona.com/doc/percona-toolkit/3.0/index.html)|[Github Percona Toolkit](https://github.com/percona/percona-toolkit)|
-| MongoDB exporter  | 0.37.0    | |[Github MongoDB Exporter](https://github.com/percona/mongodb_exporter)|
-| MySQL exporter| v0.14.0*   | [MySQL Server Exporter Documentation](https://grafana.com/oss/prometheus/exporters/mysql-exporter/)|[Github MySQL Server Exporter](https://github.com/percona/mysqld_exporter)    |
-| PostgreSQL exporter| v0.14.0*   |            |[Github PostgreSQL Server Exporter](https://github.com/percona/postgres_exporter)    |
+| PMM client/server component | Version  | Documentation |Location on GitHub|
+|-----------------------------|----------|---------------|------------------|
+| Grafana  | 11.6.0*  | [Grafana documentation](https://grafana.com/docs/grafana/latest/)|[Github Grafana](https://github.com/percona-platform/grafana)|
```

Comment:
@catalinaadam Since we are updating to 11.6.1 due to the CVE, it should be also updated here.
```suggestion
| Grafana  | 11.6.1*  | [Grafana documentation](https://grafana.com/docs/grafana/latest/)|[Github Grafana](https://github.com/percona-platform/grafana)|
```